src/lib/search-and-replace.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import util from 'util';
+import chalk from 'chalk';
+import debugLib from 'debug';
+import suffix from 'suffix';
+import { replace } from '@automattic/vip-search-replace';
+
+/**
+ * Internal dependencies
+ */
+import { trackEvent } from 'lib/tracker';
+import { confirm } from 'lib/cli/prompt';
+import { getFileSize } from 'lib/client-file-uploader';
+
+const debug = debugLib( '@automattic/vip:lib:search-and-replace' );
+
+const flatten = arr => {
+	return arr.reduce( function( flat, toFlatten ) {
+		return flat.concat( Array.isArray( toFlatten ) ? flatten( toFlatten ) : toFlatten );
+	}, [] );
+};
+
+const inPlaceReplacement = async filename => {
```

Comment:
The fn name `inPlaceReplacement` makes me think this would do the replacement... how about:

```suggestion
const inPlaceReplacementFilenames = async filename => {
```